recipes/cad1/task1/baseline/enhance.py

```diff
@@ -138,6 +138,7 @@ def map_to_dict(sources: np.ndarray, sources_list: List[str]) -> Dict:
     return signal_stems
 
 
+# pylint: disable=unused-argument
```

Comment:
The `left_audiogram` and `right_audiogram` are the cause of this linting error. Since they don't appear to be used in this function hey can be removed from the `def()` (and docstring).

Reply:
This was intentionally left in this way. The idea is to show participants that even thou the baseline is not using the audiograms, they can modify the function and take use of them.